x-pack/plugins/cloud_security_posture/README.md

```diff
@@ -50,14 +66,45 @@ yarn test:ftr --config x-pack/test/api_integration/config.ts
 Run [**End-to-End Tests**](https://www.elastic.co/guide/en/kibana/current/development-tests.html#_running_functional_tests):
 
 ```bash
-yarn test:ftr --config x-pack/test/cloud_security_posture_functional/config.ts --debug
+yarn test:ftr --config x-pack/test/cloud_security_posture_functional/config.ts
+yarn test:ftr --config x-pack/test/api_integration/config.ts --include-tag=cloud_security_posture
+yarn test:ftr --config x-pack/test/cloud_security_posture_api/config.ts
+yarn test:ftr --config x-pack/test_serverless/api_integration/test_suites/security/config.ts --include-tag=cloud_security_posture
```

Comment:
fyi, `include-tag` doesn't apply when using `test:ftr` 🤷 

it only worked when running the server and the runner separately. Please double check that it works

Reply:
Checked that, it does work, at least for servelress tests. For all the rest we have dedicated config files anyway. While checking, discovered that the command for running ESS/stateful api ftr test was incorrect, fixing it here together with missing fleet headers https://github.com/elastic/kibana/pull/168503/files